src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx

```diff
@@ -13,20 +14,96 @@ const TermsOfUsePage: React.FC = () => {
       case 'en':
         return (
           <>
-            <h1>Terms of use</h1>
+            <h1>Data protection and terms of use</h1>
```

Comment:
Looks like there are unit tests for checking these headings in https://github.com/City-of-Helsinki/linkedcomponents-ui/blob/develop/src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx#L525. Should change the titles here 🙂

Reply:
Updated test datas